windows/install.cmd

```diff
@@ -0,0 +1,115 @@
+@ECHO OFF
+SETLOCAL
+
+SET MS_EMOJI_FONT_PATH="%SystemRoot%\Fonts\seguiemj.ttf"
+SET MS_FONT_PATH="%SystemRoot%\Fonts\seguisym.ttf"
+SET EMOJI_FONT_PATH="%CD%\EmojiOneColor-SVGinOT.ttf"
+SET FINAL_EMJ_FONT_PATH_NO_QUOTES=%CD%\Segoe UI Emoji with EmojiOne.ttf
+SET FINAL_EMJ_FONT_PATH="%FINAL_EMJ_FONT_PATH_NO_QUOTES%"
+SET FINAL_FONT_PATH_NO_QUOTES=%CD%\Segoe UI Symbol with EmojiOne.ttf
+SET FINAL_FONT_PATH="%FINAL_FONT_PATH_NO_QUOTES%"
+
+ECHO Checking if Segoe UI Emoji is installed
+
+REM Windows 8 uses Segoe UI Emoji in addition to Symbol
+REM Windows 7 only uses Segoe UI Symbol
+REM We have to replace _both_ 
+ECHO Checking if Segoe UI Symbol is installed.
+
+IF NOT EXIST %MS_FONT_PATH% (
+    ECHO.
+    ECHO You don't seem to have the Segoe UI Symbol Font installed.
+    ECHO https://support.microsoft.com/en-us/kb/2729094
+    GOTO :ERROR
+)
+
+ECHO Checking if prerequisites are installed.
+
+WHERE python /q || (
+    ECHO.
+    ECHO Python.exe not found, install or add to PATH.
+    ECHO.
+    GOTO :ERROR
+)
+
+WHERE pip /q || (
+    ECHO.
+    ECHO Pip.exe not found, install or add to PATH
+    ECHO.
+    GOTO :ERROR
+)
+
+ECHO Ensuring the latest FontTools is installed.
+
+pip install --upgrade https://github.com/behdad/fonttools/archive/master.zip
```

Comment:
Can you add a call to `WHERE` for `ttx`? Might as well just check that after the `fonttools` install, confirms everything is working.

Otherwise this is looking really good! Thank you